ConsoleApp/School/Class.cs

```diff
@@ -0,0 +1,9 @@
+namespace ConsoleApp.School;
+
+class Class
+{
+    private int _classId { get; set; }
```

Comment:
not critical now, but code convention for the properties are always PascalCase
so it should be
```suggestion
    private int ClassId { get; set; }
```